packages/ts-api-react-forms/src/schema.ts

```diff
@@ -5,3 +5,12 @@ export const createTeamFormSchema = z.object({
     .string({ required_error: "Team name is required" })
     .min(1, { message: "Team name is required" }),
 });
+
+export const teamAddMemberFormSchema = z.object({
+  user: z
+    .string({ required_error: "Username is required" })
+    .min(1, { message: "Username is required" }),
+  role: z
```

Comment:
Checking the length for a select makes little sense. And I think we have a default value so even the required-check is probably not needed (although I guess a completionist might leave the latter just to be sure).